www/src/App.tsx

```diff
@@ -1,21 +1,54 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+// Libraries
+import React, { FC } from 'react';
+import { Route, BrowserRouter as Router, Switch } from 'react-router-dom';
 
-function App(): JSX.Element {
+// Material UI
+import { Container, CssBaseline, ThemeProvider } from '@material-ui/core';
+
+// Components and Routes
+import { Header, Section } from './components/Header';
+import { Landing } from './routes/Landing';
+import { ResumeReview } from './routes/ResumeReview';
+import { MockInterview } from './routes/MockInterview';
+import { Community } from './routes/Community';
+
+// Styles
+import theme from './styles/theme';
+
+// Constants
+import { COMMUNITY, COMPE_PLUS, MOCK_INTERVIEW, RESUME_REVIEW } from './util/constants';
+
+const header_sections: Section[] = [
+    { title: RESUME_REVIEW, url: '/resume-review' },
+    { title: MOCK_INTERVIEW, url: '/mock-interview' },
+    { title: COMMUNITY, url: '/community' },
```

Comment:
Should the route urls be cosntants to? Or in other words, are we going to be using these hardcoded urls in other places too?

Reply:
I was thinking that too. At the moment I can't think of other places they could be used, but it wouldn't hurt to put these route urls in constants I think, just in case.